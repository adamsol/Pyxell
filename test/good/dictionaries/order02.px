
d = {for i in 1..100 yield (-i, i): 1/i}

for p, i in d, 1..100 do
    if p != ((-i, i), 1/i) do
        print false

print {...:d by -20}
