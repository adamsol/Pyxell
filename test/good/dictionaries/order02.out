{(-100, 100): 0.01, (-80, 80): 0.0125, (-60, 60): 0.0166667, (-40, 40): 0.025, (-20, 20): 0.05}
